src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -100,6 +124,19 @@ export class TechRecordSummaryComponent implements OnInit {
     this.store.dispatch(updateEditingTechRecord({ techRecord: this.vehicleTechRecordCalculated }));
   }
 
+  findAxleToRemove(axles: Axle[]): number {
+    let previousAxleRow = 1;
+
+    for (const ax of axles) {
+      if (ax.axleNumber === previousAxleRow) {
+        previousAxleRow += 1;
+      } else {
+        return ax.axleNumber! - 1;
+      }
+    }
+    return axles.length + 1;
+  }
+
```

Comment:
This function is confusing to me, but it was already there so yeah why not

Reply:
This can be made super clean using Sets :D